### PR TITLE
#5106 Bump NodeJS and Yarn versions

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #5106)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "engines": {
-    "node": "^10.16.3"
+    "node": ">=10.16.3 <13 "
```
Comment (konekoya) [inline ohara-manager/client/package.json:6]: The next support version for Manager is nodeJS 12, i will keep it compatible until 10 is no longer maintained which is expected to be Apr 2021 :)

Comment (saivirtue) [reply ohara-manager/client/package.json:6]: Be with it! 😄

File: docker/deps.dockerfile [hunk 2]
```diff
@@ -32,12 +32,12 @@ RUN yum install -y \
...
 # install nodejs
-# NOTED: ohara-manager requires nodejs 10.x
-RUN curl -sL https://rpm.nodesource.com/setup_10.x | bash -
+# NOTED: ohara-manager requires nodejs 12.x
+RUN curl -sL https://rpm.nodesource.com/setup_12.x | bash -
 RUN yum install -y nodejs
 
 # install yarn
-RUN npm install -g yarn@1.17.3
+RUN npm install -g yarn@1.12.4
```
Comment (konekoya) [inline docker/deps.dockerfile:40]: @chia7712 both NodeJS and Yarn are updated :)

Comment (chia7712) [reply docker/deps.dockerfile:40]: Did you verify the dockerfile on your local?

Comment (konekoya) [reply docker/deps.dockerfile:40]: Yep, works fine. I will push both "manager" and "deps" images to docker hub later on

**Comment by saivirtue** [comment]
Sorry for the conflicting 😨

**Comment by konekoya** [comment]
> Sorry for the conflicting

No problem XD

**Comment by konekoya** [comment]
retry

**Comment by konekoya** [comment]
retry manager-e2e

**Pull request merged**